packages/compass-saved-aggregations-queries/src/components/saved-item-card.tsx

```diff
@@ -45,7 +53,7 @@ const NamespacePart: React.FunctionComponent<{
         glyph={type === 'database' ? 'Database' : 'Folder'}
         color={uiColors.gray.dark1}
```

Comment:
You'll need this though to not allow icon to change size

```suggestion
        color={uiColors.gray.dark1}
        className={css({flex: 'none'})}
```